### PR TITLE
include project requirements

### PR DESCRIPTION
**Pull request by Gnwin**
## Vet-Clinic-Database: database performance audit

This project is about SQL and databases.
- A database and table was created and queried.
- Table was altered.
- More queries and sql transactions on the created table.
- More queries including joins.
- More add join tables where added.
- More tables and a performance audit(EXPLAIN ANALYZE) was implemented.

All requirements for this project have been fulfilled according to this document:

https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md

With my PC, the execution time was less than a 1000ms. There was no need for decreasing execution time.

Screenshots:

Query 1: 
Screenshot 2022-06-21 at 2 18 22 PM

Query 2: 
Screenshot 2022-06-21 at 2 49 26 PM

Query 3: 
Screenshot 2022-06-21 at 2 50 39 PM

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -168,3 +168,7 @@ JOIN animals ON visits.animal_id = animals.id
 JOIN species ON animals.species_id = species.id
 WHERE vets.name = 'Maisy Smith'
 GROUP BY species.name;
+
+-- database performance audit -------------------------------------
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
```
Comment (dasileker) [inline queries.sql]: - [x] Hello and nice work on the analyzer, but as you have seen that you are giving 3 challenges ` SELECT COUNT(*) FROM visits where animal_id = 4; **and** SELECT * FROM visits where vet_id = 2, **and** SELECT * FROM owners where email = 'owner_18327@mail.com';`

File: schema.sql [hunk 2]
```diff
@@ -76,3 +76,7 @@ CREATE TABLE visits (
   vet_id integer REFERENCES vets(id),
   visit_date date
 );
+
+-- database performance audit -------------------------------------
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
```
Comment (dasileker) [inline schema.sql]: - [x] Also you have missed a requirement the **```CREATE INDEX ON visits (animal_id); also => CREATE INDEX ON visits (vet_id); also => CREATE INDEX ON owners (email);```**

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Gnwin,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**
- PR is very descriptive 👍.
- The readme file is also very descriptive 👍.
- The database code is clean 👌.

## Required Changes ♻️
_Check the comments under the review._
- Hello and fantastic work on the vet-clinic-DB however, you have missed some requirements, please ensure to add them since your DB is getting the right results it will not be an issue for you to add them.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @Gnwin, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- [x] Please make sure to apply the changes the previous reviewer asked for before submitting a new review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 2 above)
Comment (Gnwin) [reply schema.sql]: I  am not seeing this missed requirement in the document

Comment (dasileker) [reply schema.sql]: Hello as I mentioned the index create is not added to your schema

File: queries.sql (hunk 1 above)
Comment (Gnwin) [reply queries.sql]: I implemented all three challenges. The three challenges are also in my project in queries sql file. The screenshots are in the PR.

File: schema.sql (hunk 2 above)
Comment (Gnwin) [reply schema.sql]: Its done. thanks @dasileker

**Review by medaminedev66 — APPROVED** [review]
Hi @Gnwin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 💯  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**